layouts/partials/footer.hbs

```diff
@@ -3,10 +3,10 @@
 <footer {{#if className}} class="{{className}}"{{/if}}>
 
   <div class="container">
-    <div class="linuxfoundation-footer">
+    <div class="openjsfoundation-footer">
       <div class="issue-link-container">
-        <a class="linuxfoundation-logo" href="http://collabprojects.linuxfoundation.org">
-          <img alt="Linux Foundation Collaborative Projects" src="/static/images/lfcp.png" width="416" height="15">
+        <a class="openjsfoundation-logo" href="https://openjsf.org/">
+          <img alt="OpenJS Foundation" src="/static/images/openjs_foundation-logo.png" width="200">
```

Comment:
Please add the height too

Reply:
I was about to switch to the SVG logo here so we might as well do it in this branch. No need to even add a PNG IMHO